### PR TITLE
*: Add leak test

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: store/localstore/kv.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 PingCAP, Inc.
+// Copyright 2015 PingCAP, Inc.ctory.
```
Comment (qiuyesuifeng) [inline store/localstore/kv.go]: Remve this bad coment.

File: util/testleak/add-leaktest.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+# Usage: add-leaktest.sh pkg/*_test.go
+
+set -eu
+
+sed -i'~' -e '
```
Comment (qiuyesuifeng) [inline util/testleak/add-leaktest.sh]: Have u tested in `MAC OSX`, i remember that the `sed` command is not the same with `*nix`.

**Comment by qiuyesuifeng** [comment]
@ngaut PTAL

File: util/testleak/add-leaktest.sh (hunk 2 above)
Comment (zimulala) [inline util/testleak/add-leaktest.sh]: I have tested in MAC OSX.

File: util/testleak/add-leaktest.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@ sed -i'~' -e '
   /^func (s \*test.*Suite) Test.*(c \*C) {/ {
     n
     /testleak.AfterTest/! i\
-	defer testleak.AfterTest(c)()
+		defer testleak.AfterTest(c)()
```
Comment (qiuyesuifeng) [inline util/testleak/add-leaktest.sh]: More than one tabs?

**Comment by shenli** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
PTAL @qiuyesuifeng @ngaut

**Comment by ngaut** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**